### DISCUSSION TITLE
How to remove item from a scalar list

### DISCUSSION THREAD

**Discussion by veritem** [prisma/prisma#7386] [Q&A] [answered]
I have a scalar ist of string using Postgres database and I'm not able to remove items from the scalar item, I always get update error, is there any way to update scalar list

**Comment by ryands17** [comment, marked as answer]
@makuzaverite 👋 

Currently the only way to remove items is to fetch the value first and then remove the item needed from the list. I would suggest creating a feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by veritem** [reply]
Thanks for the reply

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
